src/super_gradients/training/processing/processing.py

```diff
@@ -357,6 +371,162 @@ def preprocess_image(self, image: np.ndarray) -> Tuple[np.ndarray, None]:
         return cropped_image, None
 
 
+@register_processing(Processings.SegResizeWithPadding)
+class SegResizeWithPadding(Processing, ABC):
```

Comment:
Why ABC? Is that a mistake ?

Reply:
fixed (b55fcd97f9cf830a80fe7a9ed00aec6999fb374a)